paddle/fluid/pybind/pir.cc

```diff
@@ -223,6 +224,20 @@ std::string GetValueInfo(Value v) {
   return ss.str();
 }
 
+Value GetOutputValueByName(const Program &program, const std::string &name) {
+  auto &block = *program.block();
+  pir::StrAttribute name_attr =
+      pir::StrAttribute::get(IrContext::Instance(), name);
+  for (auto &op : block) {
+    if (op.isa<pir::ShadowOutputOp>()) {
+      if (op.attribute("output_name") == name_attr) {
```

Comment:
we need to assert the name is unique

Reply:
very thanks, I will fix the problem in next pr.